docs/guides/_toc.json

```diff
@@ -302,7 +302,7 @@
               "title": "Configure runtime options",
               "children": [
                 {
-                  "title": "Configure runtime compilation",
+                  "title": "Configure error suppression",
```

Comment:
Yeah; I left out Runtime in the TOC because it's in the Runtime section, as you pointed out.  But I left "runtime" in the page title for context when searching or when you're linked to the page from elsewhere.

Reply:
It's tricky bc I once got a complaint from Jay that the title of the page did not match what was in the toc. But long toc names are noisy and unhelpful. If we have `runtime` in the metadata title/description, that should help with search results - then leave it off both the page name and toc listing - my 2 cents!